### ISSUE TITLE
Suggestion: Update / Upgrade process / Wiki Creation

### ISSUE THREAD

**Issue by mach7** [AntonOsika/gpt-engineer#161]
What would the official  / correct process be to update to the latest code that you guys are working on?

Should we just be doing a git clone?

Maybe a update.sh and/or update.bat can be made that can be ran to process an official upgrade process?

Also, what are the thoughts on making a separate branch that can be used for the development / nightly  and maybe using the main / master branch as a "stable" branch. 

Also was wondering if a wiki might be in the works??

**Comment by patillacode** [comment]
Hi @mach7 

TLDR;
```bash
$ pip install --upgrade gpt-engineer
```

> what are the thoughts on making a separate branch that can be used for the development / nightly and maybe using the main / master branch as a "stable" branch.

Here is the way we decided to approach this:

- `main` is our bleeding edge branch
- A release will be done every few days (with the latest tested changes) available in pypi

We encourage users to use the version you would get from the command above if you want a stable release.

I have made small modification to the README to better transmit this to the public, you can have peek [here](https://github.com/AntonOsika/gpt-engineer/blob/fe2028ca37cff36ee3b947491a43b1d5b96e504c/README.md#installation).
We will merge this soon.

> Maybe a update.sh and/or update.bat can be made that can be ran to process an official upgrade process?

I guess this could be added to the `Makefile`

> Also was wondering if a wiki might be in the works?

Not that I know of but please feel free to generate a new issue for that with your recommendations!

**Comment by patillacode** [comment]
If you wish to reopen the issue please do following the new issue template.
